example/app.go

```diff
@@ -53,28 +57,37 @@ func Run() error {
 		panic(err)
 	}
 
-	ethClient, err := evmClient.NewEVMClient(TestEndpoint, false, AliceKp)
+	ethCfg, err := evmClientConfig.GetConfig(".", "config")
+	if err != nil {
+		panic(err)
+	}
+	ethClient := evmClient.NewEVMClient()
+	err = ethClient.InitializeClient(ethCfg, AliceKp)
 	if err != nil {
 		panic(err)
 	}
 	evmListener := listener.NewEVMListener(ethClient)
-	evmListener.RegisterHandler("0x3167776db165D8eA0f51790CA2bbf44Db5105ADF", evmClient.HandleErc20DepositedEvent)
+	evmListener.RegisterHandlerFabric(ethCfg.Erc20Handler, ethClient.ReturnErc20HandlerFabric)
 
 	evmWriter := writer.NewWriter(ethClient)
-	evmWriter.RegisterProposalHandler("0x3167776db165D8eA0f51790CA2bbf44Db5105ADF", writer.ERC20ProposalHandler)
+	evmWriter.RegisterProposalHandler(ethCfg.Erc20Handler, writer.ERC20ProposalHandler)
 
-	evmChain := evm.NewEVMChain(evmListener, evmWriter, db, "0x62877dDCd49aD22f5eDfc6ac108e9a4b5D2bD88B", 0)
+	evmChain := evm.NewEVMChain(evmListener, evmWriter, db, ethCfg.Bridge, 0)
 	if err != nil {
 		panic(err)
 	}
-
-	kp, err := keystore.KeypairFromAddress("5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY", keystore.SubChain, "alice", true)
+	subCfg, err := subClientConfig.GetConfig(".", "subConfig")
+	if err != nil {
+		panic(err)
+	}
+	kp, err := keystore.KeypairFromAddress(subCfg.GeneralChainConfig.From, keystore.SubChain, "alice", true)
 	if err != nil {
 		panic(err)
 	}
 	krp := kp.(*sr25519.Keypair).AsKeyringPair()
 
-	subC, err := subClient.NewSubstrateClient("ws://localhost:9944", krp, stopChn)
+	subC := subClient.NewSubstrateClient()
+	err = subC.InitializeClient(subCfg.GeneralChainConfig.Endpoint, krp, stopChn)
 	if err != nil {
 		panic(err)
 	}
```

Comment:
Same comment as above; consolidate code from `InitializeClient` into `NewSubstrateClient` and pass `subCfg` as argument to `NewSubstrateClient`